internal/pkg/docker/docker.go

```diff
@@ -43,6 +44,27 @@ func (r Runner) Build(uri, imageTag, path string) error {
 	return nil
 }
 
+// BuildWithMultipleTags is the same as Build but is able to accept multiple tags.
+func (r Runner) BuildWithMultipleTags(uri, path string, imageTags []string) error {
```

Comment:
I wonder if we can just augment the previous `Build` and `Push` methods with new tags. For example, what do you think of this:
```go
func (r Runner) Build(uri, path, imgTag string, additionalTags ...string) error
func (r Runner) Push(uri, imgTag string, additionalTags ...string) error
```
This way callers know that at least one tag is required and can add additional ones if they want to:  https://dave.cheney.net/practical-go/presentations/qcon-china.html#_prefer_var_args_to_t_parameters

Reply:
Sure sure!! I wanted to create a separate PR that basically do what you suggested^ after this series of `task run` PRs. Just wanted to make sure that people are okay with these before I touch other parts of the codebase, e.g. `svc`. But definitely I can do that in this PR!!